### DISCUSSION TITLE
Medical Model API?

### DISCUSSION THREAD

**Discussion by rmappillai** [deepgram/community#376] [General help] [answered]
Hi,

I'm in touch with Nick, your product consultants, who has confirmed that my project (590992eb-8df5-48ef-b6fc-0ed08db8e5ba) has access to the medical model and nova-2. I've been successfully invoking the nova, nova-2-ea, whisper, whisper-large models. On calling model: "2-ea-medical" or "medical-2-ea" the I get back the 400 Bad Request response. Same goes when I try model "nova2" or "nova-2". Access to my project has been confirmed, so I'm not sure why I get 400 Bad Request when invoking a working api call using 2-ea-medical or nova2? Any suggestions? (screenshot below demonstrates 400 unsuccessful 2-ea-medical call followed by successful nova-2-ea call).

![image](https://github.com/deepgram/community/assets/145132418/6efd2611-46eb-4322-b8f3-a83c9e27884c)

**Comment by jjmaldonis** [comment]
Hey @rmappillai, try using `model=medical-2-ea` rather than `model=2-ea-medical`. That should fix the issue.

**Reply by rmappillai** [reply]
Thanks @jjmaldonis 

I get insufficient permissions? But Nick has assured me that he double checked and my project has permissions?

<img width="1470" alt="image" src="https://github.com/deepgram/community/assets/145132418/f6fd1a63-acee-4da2-bcb5-d9d29679ce65">

**Reply by jjmaldonis** [reply, marked as answer]
@rmappillai I see what it is now. Please use the same model name as you had before: `model=2-ea-medical`, and add `tier=nova` to your request.

So your full request parameters should be: `?smart_format=true&model=2-ea-medical&tier=nova`

**Reply by rmappillai** [reply]
Thanks @jjmaldonis - this worked. I'm analyzing the results. Looks really good on first glance (except for the date of birth which it outputted as: "tentwelvenineteen 90" unlike nova which gave out "10/12/1990" - any setting I can give to fix this? Thanks again!
